### DISCUSSION TITLE
Grid name feature

### DISCUSSION THREAD

**Discussion by adnovea** [slipx06/sunsynk-power-flow-card#229] [Ideas]
This add-ons is amazing and get new valuable features very often. Thanks slipx06 for your excellent work.

May I suggest to add a "**grid_name**" option in the grid section (_e.g. default can be replaced by the energy provider name_)
Almost every parts can be customized but the Grid name so far.
Moreover, the text is a _bit too close_ to the grid icon.

These are not mandatory changes, just some ideas for the future.

**Comment by slipx06** [comment]
Good idea. Thanks

**Comment by slipx06** [comment]
I've added this to [4.13.0](https://github.com/slipx06/sunsynk-power-flow-card/releases/tag/v4.13.0)

**Comment by adnovea** [comment]
THANKS, l will update as soon as the version will be released.
I really appreciate this very detail and easy-to-use add-on to manage my system on a day-to-day basis .

I wrote a fork of the Solarman add-on for LSE-3 because Ethernet stick logger were not supported.
https://github.com/adnovea/solarman_lse

I also poorly tried to mimic your project for my Heat-pump and my DHW Solar-boiler but they are lower quality regarding your add-on and dedicated for only one type of equipment without support for translation.
I don't have the framework for developing so I wrote them in raw JS+HTML.
Your add-on idea is great ;-)

![image](https://github.com/slipx06/sunsynk-power-flow-card/assets/44359861/3cf54acd-3911-46d5-9baa-c811f9bcfe12)
It can support DHW on HP with 3-way valve.

![image](https://github.com/slipx06/sunsynk-power-flow-card/assets/44359861/76fa5426-cec8-4890-8b80-c99af2508163)

**Comment by slipx06** [comment]
Looks great. Well done. Every project has a beginning.  This was the very first version of my card 😄 

![image](https://github.com/slipx06/sunsynk-power-flow-card/assets/7227275/fefd3031-f621-4e7d-a107-6d62098dc812)

**Comment by adnovea** [comment]
Your add-on for my system ... Perfect
![image](https://github.com/slipx06/sunsynk-power-flow-card/assets/44359861/2b380f53-b2cd-491f-a8c6-5ad63b54dacb)

_Just a little overlap on AutoConso and Ratio_

**Reply by slipx06** [reply]
There are some issues when translating to longer text. Let me see if I can move Ratio over to the right
